lib/puppet_pal.rb

```diff
@@ -54,7 +54,7 @@ def call_function(function_name, *args, &block)
       internal_evaluator.evaluator.external_call_function(function_name, args, topscope, &block)
     end
 
-    # Returns an Puppet::Pal::FunctionSignature objects or nil if function is not found
+    # Returns a Puppet::Pal::FunctionSignature objects or nil if function is not found
```

Comment:
Should say `object` instead of `objects`. Also, the actual `@return` tag further down claims that it indeed is an array.